dev-libs/litehtml/litehtml-0.9.recipe

```diff
@@ -0,0 +1,76 @@
+SUMMARY="Lightweight HTML/CSS rendering engine"
+DESCRIPTION="
+Litehtml is lightweight HTML/CSS rendering engine. The main goal of the \
+litehtml library is to give the developers the easy way to show the HTML pages \
+in theirs applications."
+HOMEPAGE="http://www.litehtml.com/"
+COPYRIGHT="2013-2025 Aliaksei Chapyzhenka"
+LICENSE="BSD (2-clause)"
+REVISION="1"
+SOURCE_URI="https://github.com/litehtml/litehtml/archive/refs/tags/v0.9.tar.gz"
+CHECKSUM_SHA256="ef957307da15b1258a70961942840bcf54225a8d75315dcbc156186eba35b1a7"
+PATCHES="litehtml-0.9.patchset"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+libVersion="0.0.0"
+libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
+
+PROVIDES="
+	litehtml$secondaryArchSuffix = $portVersion
+	lib:libgumbo$secondaryArchSuffix
+	lib:liblitehtml$secondaryArchSuffix
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	litehtml${secondaryArchSuffix}_devel = $portVersion
+	devel:libgumbo$secondaryArchSuffix
+	devel:liblitehtml$secondaryArchSuffix
+	"
+REQUIRES_devel="
+	litehtml$secondaryArchSuffix == $portVersion base
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:cmake
+	cmd:gcc$secondaryArchSuffix
+	cmd:ld$secondaryArchSuffix
+	cmd:make
+	"
+
+BUILD()
+{
+	cmake -Bbuild -S. -DCMAKE_BUILD_TYPE=Release \
+		$cmakeDirArgs \
+		-DBUILD_SHARED_LIBS=ON \
+		-DLITEHTML_BUILD_TESTING=OFF
+
+	make -C build $jobArgs
+}
+
+INSTALL()
+{
+	make -C build install
+
+	# copy the container for Haiku
```

Comment:
Not famaliar with the containers, are these required?

Reply:
Looking at Arch package list I don't see them showing up?
https://archlinux.org/packages/extra/x86_64/litehtml/

Reply:
You're right the container part is not included. It means it must be added in each implementation once the librairies is included in a project.